### ISSUE TITLE
Suggestion: less changing of data files?

### ISSUE THREAD

**Issue by diyadas** [swcarpentry/shell-novice#802]
I taught this lesson yesterday and we spent a lot of time changing data from episode to episode:

- episode 2: north-pacific-gyre/2012-07-03/
- episode 3:
  - create thesis directory
  - two examples of data/filesystems that learners don't have in front of them
- episode 4: This is the one that was utterly confusing.
  - molecules
  - data directory for one exercise
  - molecules
  - data directory for another few exercises
  - north-pacific-gyre/2012-07-03
- episode 5:
  - creatures
  - an exercise using molecules
  - creatures for an example
  - an exercise using molecules
  - creatures
  - north-pacific-gyre/2012-07-03
  - exercises using molecules
- episode 6:
  - molecules
  - exercise using data/animal-counts/
  - north-pacific-gyre/2012-07-03
  - exercise using data-shell/molecules
  - exercise using north-pacific-gyre/2012-07-03

I understand that there is value in 
1. demonstrating how to change directories repeatedly
2. showing learners that these functions can be applied to many types of data

but I think this might underestimate the cognitive load of maintaining an understanding what types of data are in each file - what was the experiment conducted. I spent a lot of time reintroducing data sets rather than functions.

Maybe each episode could pick a single dataset to focus on? That way we do change directories and work with different types of data, but also have the opportunity to focus on the functions within episodes.

**Comment by r-barnes** [comment]
I taught part of the same lesson and whole-heartedly agree with @diyadas 's comments.

In addition to her thoughts, I find the organization of the directory a bit ridiculous in a workshop in which we're teaching good practices. The main directory contains `data/` there's also `writing/data`. `molecules` and `creatures` both contain data, but are not labeled as such.

My prefered organization would be to have a structure more like:

    projects/
      north-pacific-gyre/
        writing/
        data/
          2012-07-03/
      biodiversity/
        data/
          basilisk.dat
          unicorn.dat
      seminar-reading/
        solar.pdf

Perhaps other projects could be added to capture the types of data used by other disciplines. I feel like the foregoing is a defensible organization. There could even be a brief section were we reorganize Nelle's files so they're better organized.

Also note that `animals.txt` should be `animals.csv`, `planets.txt` should be `planets.csv`, and `sunspot.txt` should probably have an extension as well.

**Comment by gcapes** [comment]
Many good points raised above.
In addition to organising the files better, there are a number of unused files which I would be in favour of deleting:

- notes.txt
- pizza.cfg
- solar.pdf

in the root directory, and others in various subdirectories.

**Comment by fdavenport** [comment]
I taught this lesson yesterday and had the same reaction. The main story line (Nelle's analysis) was sometimes buried pretty deep in within each episode. As mentioned above, if you aren't following the lesson exactly, these examples are hard to keep track of and easily lost. I think the lesson would be a lot more effective for some students if the examples focus more on Nelle's research pipeline and use the other files and directories for more secondary exercises.

**Comment by kcranston** [comment]
I agree completely with the sentiment here. I just taught this lesson this morning, and the jumping around within the data-shell directory without any consistent narrative was extremely jarring. I ended up skipping the Nelle narrative completely (since the episodes tend to introduce a command with the non-Nelle files and that's all I had time for; see #928 about timing overall).

Specific recommendations:
* clean up the data-shell directory, removing unused files
* remove everything but the Nelle data and exercises in order to focus on the story of operations on that data

**Comment by colinmorris** [comment]
We have an old issue which is closely related: #371 *Audit filesystem/lesson examples for consistency*

The issue of the coherence of the Nelle framing narrative is something that's come up a lot (I thought we might have an issue tracking it, but I couldn't find one). Unfortunately, fixing it is a huge task which is by nature difficult to work on in a piecemeal fashion. 

I'd be curious how other Carpentry lessons handle this issue. Are there others that do a better job of maintaining a running narrative - and if so, how? Do many lessons just not bother with one, and instead choose to keep each episode independent and modular? (The only other lesson I have a passing familiarity with is the version control one. Last time I taught it, I believe it had some kind of Universal Classic Monsters running motif, but it felt highly vestigial, even compared to Nelle.)

**Comment by diyadas** [comment]
Hi 👋 @swcarpentry/r-novice-inflammation-maintainers here

I think like this seems like an appropriate use case for a draft PR, possibly in consultation with the curriculum advisory committee?

**Comment by kekoziar** [comment]
I just want to address the changing folders aspect of this, not the bad example of file/folder names:

When I first taught the lesson, I thought it was pretty ridiculous to change file folders a lot.  But, then I remembered that I am an advanced user of Bash.  The lesson provides good practice for novice learners on changing file folders. I think that's the point: to connect the muscle memory with the conceptual framework of folder organization and gain confidence on navigating the shell.

**Comment by deppen8** [comment]
Is this addressed by #1233 ?
